R/tf.R

```diff
@@ -41,6 +42,8 @@ tf <-
     return(obj)
   }
 
+
+#' @export
```

Comment:
These are all S3 methods

Reply:
I assume these can still be called with `update(tf, ...)` etc even if not exported? @josephguillaume, would you rather they weren't exported?

Reply:
In the NAMESPACE file they need to appear as S3 method definitions rather than exports. I think what @mortisflux said is that export is clever enough to do this.